src/services/agent-service.ts

```diff
@@ -130,7 +130,7 @@ export class AgentService {
     resources = resources.concat(
       this.snapshotService.buildLogResource(snapshotLog),
       // @ts-ignore we won't write anything if css is not is passed
-      this.snapshotService.buildPercyCSSResource(percyCSSFileName, snapshotOptions.percyCSS),
+      this.snapshotService.buildPercyCSSResource(percyCSSFileName, snapshotOptions.percyCSS, snapshotLogger),
```

Comment:
The log has already been severed from the logger at this point. Move this line to before `buildLogResource` so they are still attached when you log inside the method.

Reply:
Ah, right. Good catch